source/jsonnet/england-wales/household/blocks/who-lives-here/do_you_usually_live_here.jsonnet

```diff
@@ -85,7 +85,7 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
     ],
     answers: [
       {
-        id: 'you-live-here-answer',
+        id: 'do-you-usually-live-here-answer',
```

Comment:
This answer id is now out of sync with its parent `primary-confirmation-question`?